### ISSUE TITLE
[BUG] [GPU] Avoid multiple `system_profiler` log display

### ISSUE THREAD

**Issue by sivan** [HorlogeSkynet/archey4#104]
I'm using a MacPro 2013 with two Graphic Cards, archey always return a log before the output, just like below:

```
2021-11-10 16:48:16.372 system_profiler[32818:899967] Device PreExisted [0000000100000802] AMD Radeon HD - FirePro D500
2021-11-10 16:48:16.372 system_profiler[32818:899967] Device PreExisted [00000001000007e8] AMD Radeon HD - FirePro D500
```

How can I remove this log or any option to ignore this?

**Comment by HorlogeSkynet** [comment]
Hey @sivan, it looks like `system_profiler` is writing some output to STDERR, I'll silence this soon.
Bye 👋

---

Related issue : jlhonora/lsusb#5

**Comment by HorlogeSkynet** [comment]
Just released as [v4.13.3](https://github.com/HorlogeSkynet/archey4/releases/tag/v4.13.3), on its way to Homebrew : Homebrew/homebrew-core#89334

Bye :wave:
